modules/ROOT/pages/example-einstein-prompts.adoc

```diff
@@ -0,0 +1,102 @@
+        "class": <replace with vehicle classification if found">,
+        "eye_color": <replace with eye color if found">,
+        "sex": <sex as specified>,
+        "weight": <replace with the weight found>,
+        "restrictions":<replace with any restrictions listed in the document>,
+        "address": {
+            "street": <replace with the street line of the address including any apartment numbers>,
+            "city":<replace with the city in the address found>,
+            "state": <replace with the state found in the address>,
+            "zip": <replace with the zipcode found in the address> 
+        }
+    }
+}
+----
+
+== Classify a Document With Multiple Pages
+
+In this example, the source document is a PDF with multiple pages containing a timesheet summary document with multiple entries. The goal is to classify the content by identifying the name of the person in each page. For such a case, the following prompt returns a list in JSON format containing the name of the person and the corresponding page number: 
+
+[source]
+----
+Classify each page of the following timesheet summary as a unique document type and also identify whether the page continues from the previous document type or it is a new one. 
+Summarize all the employee's names in the timesheet summary document and the pages where their timesheet entry is found. 
+Structure the output as a JSON where the employee name is the key and the page number is the value.
+----
+
+== Extract Form Data as Key/Value Pairs 
+
+The following example prompt analyzes a form and extracts all data as key value pairs in JSON format: 
+
+[source]
+----
+Extract important information in the text delimited by *********
+The data in the text is a medical form and has key/value pairs. 
+Extract and present the key value pairs in a JSON formatted response.
+
+Do not create new information, interpret only the information in the provided document.
+
+An example output JSON is:
+    
+    "patient_name: {
+        "value" : "Max the Mule",
+    }
+----
+
+== Extract Form Data as Key/Value Pairs Including Handwritten Values
+
+The following example is a prompt that analyzes a medical certificate to extract key/value pairs, assigns a confidence score to each value, and also considers handwriting during the extraction: 
+
+[source]
+----
+Extract important information from the document.
+Present the key value pairs in a JSON formatted response.
+
+If you find a date, then format it in yyyy/mm/dd format.
+If you find an age, simply return the number and drop the "years old".
+If you see a checkmark on a checkbox, extract that value as "Yes", otherwise, mark that option as "No". 
+
+Do not create any facts of information, interpret only the information in the document.
+For each key, value that you find, try to associate a confidence score.
+
+To generate a confidence score:
+If you are fairly confident about the information in the text, then show confidence score between 80% to 90%
```

Comment:
```suggestion
If you are confident about the information in the text, show a confidence score between 80% to 90%
```

Reply:
I'd hesitate to use the word "fairly", because it has multiple meanings (writing for a global audience). You might use "reasonably" or another more specific synonym if it really needs to be qualified.